.gulp/regeneration.iced

```diff
@@ -42,14 +42,17 @@ regenExpected = (opts,done) ->
     if (opts.flatteningThreshold)
       args.push("--#{opts.language}.payload-flattening-threshold=#{opts.flatteningThreshold}")
 
-    if (opts.generatePackageJson)
-      args.push("--#{opts.language}.generate-package-json=true")
+    if (opts.generatePackageJson != undefined && opts.generatePackageJson != null)
```

Comment:
Was the truthy check causing a problem?

Reply:
If you want to get as strict as possible with the checks, I suggest `!==` on both or just `!= null` [which returns true for both undefined and null](https://stackoverflow.com/questions/28975896/is-there-a-dedicated-function-to-check-null-and-undefined-in-typescript#28984306). (Unless Iced CoffeeScript changed != to have !== behavior?)

Reply:
I wanted to be able to pass an empty string as a value, and an empty string is falsy. This solved that problem.